YTTWu/Kusa
# Friends list mongo db

## Overview
Write a small summary of what is being changed.


## Materials
- [Jira ticket](https://meowsic.atlassian.net/jira/software/projects/GM/boards/1)
- [Applicable mockups]()


## Pictures
Provide a picture or video of your changes if applicable.

## Manual Testing Guide
Provide details of how to manually test your changes if applicable.
- Navigate to the big green button and click it.
- Note that the button plays a sound.

Note that pull requests require at least one approving review to merge.
